aconcli/cmd/restart.go

```diff
@@ -12,10 +12,13 @@ import (
 )
 
 var restartCmd = &cobra.Command{
-	Use:   "restart Container_ID",
+	Use:   "restart",
```

Comment:
Is `<container-id>` still needed?

Reply:
Not any more. container is now specified by the flag